core/templates/dev/head/library/library.html

```diff
@@ -45,9 +45,9 @@
         </div>
 
         <div class="oppia-exp-summary-tiles-container" ng-if="!inSearchMode">
+        <img src="/images/library/library_banner.svg" width="100%" align="center">
```

Comment:
Also suggest the path /images/library/banner.svg. The duplicate 'library' in the pathname seems redundant.